R/argument_conventions.R

```diff
@@ -122,6 +122,11 @@ gen_args <- function(df, alt_counts_df, spl, pos, tt, tr, verbose, colwidths, ob
 #'   uniqueness.
 #' @param vars (`character`)\cr vector of variable names.
 #' @param var_labels (`character`)\cr vector of labels for one or more variables.
+#' @param show_colcounts (`logical(1)`)\cr Should column counts be displayed at the level
```

Comment:
```suggestion
#' @param show_colcounts (`flag`)\cr Should column counts be displayed at the level
```

Reply:
flag is not a class in R, it doesn't make sense for it to be treated as such (ie surrouned in ticks)

Reply:
Fine with me. I do not have a strong opinion about this